src/components/elementView/detailedView/assetView/barcodeView.tsx

```diff
@@ -38,6 +39,10 @@ class BarcodeView extends React.PureComponent<
     );
   }
 
+  public componentWillUnmount(): void {
+    document.body.style.background = "#293742";
+  }
+
```

Comment:
Since navbar renders on printing page and the user can therefore technically navigate back to the dashboard from here, I return the background color from white to it's original state before officially navigating to another page.

Reply:
gotcha -> it might be worth just removing the navbar from this page all together but I can track that since if we do that we should probably also do it on rack print.